xsd/netex_part_1/part1_networkDescription/netex_line_support.xsd

```diff
@@ -381,4 +381,81 @@ Rail transport, Roads and Road transport
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="TypeOfServiceIdType">
```

Comment:
The first 4 elements are all defined in netex_serviceRestrictions_support.xsd => don't do those here.